### PR TITLE
fix!: remove --output flag from update commands

### LINKED ISSUE #1092: For the `update` commands, do not display the updated object as JSON by default.
Running `rhoas kafka topic update` will by default print the topic configuration to stdout. This is a massive object and it is unlikely that the user needs to see it. We should default to printing a simple success message by default, and allow users to use the output flag to output the object.

### PR DESCRIPTION
**Pull request by craicoverflow**
Closes #1092 

### Verification Steps

1. Run `rhoas kafka update`, `rhoas kafka topic update`, `rhoas kafka consumer-group reset-offset`. All will no longer show any JSON output.

### Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation change
- [ ] Other (please specify)

### Checklist

- [x] Documentation added for the feature
- [ ] CI and all relevant tests are passing
- [ ] Code Review completed
- [ ] Verified independently by reviewer

### CODE REVIEW

File: pkg/localize/locales/en/cmd/kafka.en.toml [hunk 1]
```diff
@@ -702,7 +702,7 @@ one = 'Topic "{{.Name}}" already has {{.Count}} partitions.'
 one = 'Nothing to update'
 
 [kafka.topic.update.log.info.topicUpdated]
-one = 'Topic "{{.TopicName}}" in Kafka instance "{{.InstanceName}}" has been updated'
+one = 'Topic "{{.TopicName}}" in Kafka instance "{{.InstanceName}}" has been updated. Run "rhoas kafka topic describe --name {{.TopicName}} to view its configuration."'
```
Comment (rkpattnaik780) [inline pkg/localize/locales/en/cmd/kafka.en.toml:705]:
```suggestion
one = 'Topic "{{.TopicName}}" in Kafka instance "{{.InstanceName}}" has been updated. Run "rhoas kafka topic describe --name {{.TopicName}}" to view its configuration.'
```

File: pkg/cmd/kafka/topic/update/update.go [hunk 2]
```diff
@@ -302,7 +304,7 @@ func runCmd(opts *options) error {
 	}
 
 	opts.Logger.Info(opts.localizer.MustLocalize("kafka.topic.update.log.info.topicUpdated", topicNameTmplPair, kafkaNameTmplPair))
-	return dump.Formatted(opts.IO.Out, opts.outputFormat, response)
+	return nil
```
Comment (rkpattnaik780) [inline pkg/cmd/kafka/topic/update/update.go:307]: Running update command with output flag set as yml or json have no effect. Should we remove the output flag.

**Review by rkpattnaik780 — CHANGES_REQUESTED** [review]

File: pkg/cmd/kafka/topic/update/update.go [hunk 3]
```diff
@@ -153,7 +154,6 @@ func NewUpdateTopicCommand(f *factory.Factory) *cobra.Command {
 		},
 	}
 
-	cmd.Flags().StringVarP(&opts.outputFormat, "output", "o", "json", opts.localizer.MustLocalize("kafka.topic.common.flag.output.description"))
```
Comment (wtrocki) [inline pkg/cmd/kafka/topic/update/update.go:156]: Flag was removed but variables and validation is still present. Functionally it works but leaves dead code.

File: pkg/cmd/kafka/update/update.go [hunk 4]
```diff
@@ -119,7 +118,6 @@ func NewUpdateCommand(f *factory.Factory) *cobra.Command {
 		},
 	}
 
-	cmd.Flags().StringVarP(&opts.outputFormat, "output", "o", dump.JSONFormat, opts.localizer.MustLocalize("kafka.common.flag.output.description"))
```
Comment (wtrocki) [inline pkg/cmd/kafka/update/update.go:122]: Similar. What is the reason for leaving outputFormat in options?

Comment (craicoverflow) [reply pkg/cmd/kafka/update/update.go:122]: I just forgot tbh. Will remove.

**Review by wtrocki — CHANGES_REQUESTED** [review]
Requires minimal cleanup. Verified

**Review by wtrocki — APPROVED** [review]

**Pull request merged**